networking/hub-region.v1.1.bicep

```
@@ -518,6 +518,7 @@ resource fwPolicy 'Microsoft.Network/firewallPolicies@2024-01-01' = {
               fqdnTags: []
               webCategories: []
               targetFqdns: [
+                #disable-next-line no-hardcoded-env-urls
```

I always like to leave a comment whenever there's a linter directive - maybe something like this:

```suggestion
                #disable-next-line no-hardcoded-env-urls // Disabling this rule because we explicitly want to allow Microsoft Entra ID authentication for the application, and this scenario is specific to the public Azure cloud.
```